examples/cd_model_unc_cifar10.ipynb

```diff
@@ -0,0 +1,752 @@
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "cd = ClassifierUncertaintyDrift(\n",
+    "  X_ref, model=clf, backend='tensorflow', p_val=0.05, preds_type='probs'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's check whether the detector thinks drift occurred on the different test sets and time the prediction calls:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from timeit import default_timer as timer\n",
+    "\n",
+    "labels = ['No!', 'Yes!']\n",
+    "\n",
+    "def make_predictions(cd, x_h0, x_corr, corruption):\n",
+    "    t = timer()\n",
+    "    preds = cd.predict(x_h0)\n",
+    "    dt = timer() - t\n",
+    "    print('No corruption')\n",
+    "    print('Drift? {}'.format(labels[preds['data']['is_drift']]))\n",
+    "    print('Feature-wise p-values:')\n",
+    "    print(preds['data']['p_val'])\n",
+    "    print(f'Time (s) {dt:.3f}')\n",
+    "    \n",
+    "    if isinstance(x_corr, list):\n",
+    "        for x, c in zip(x_corr, corruption):\n",
+    "            t = timer()\n",
+    "            preds = cd.predict(x)\n",
+    "            dt = timer() - t\n",
+    "            print('')\n",
+    "            print(f'Corruption type: {c}')\n",
+    "            print('Drift? {}'.format(labels[preds['data']['is_drift']]))\n",
+    "            print('Feature-wise p-values:')\n",
+    "            print(preds['data']['p_val'])\n",
+    "            print(f'Time (s) {dt:.3f}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "No corruption\n",
+      "Drift? No!\n",
+      "Feature-wise p-values:\n",
+      "[0.7868902]\n",
+      "Time (s) 31.026\n",
+      "\n",
+      "Corruption type: gaussian_noise\n",
+      "Drift? Yes!\n",
+      "Feature-wise p-values:\n",
+      "[0.]\n",
+      "Time (s) 39.606\n",
+      "\n",
+      "Corruption type: motion_blur\n",
+      "Drift? Yes!\n",
+      "Feature-wise p-values:\n",
+      "[0.]\n",
+      "Time (s) 29.787\n",
+      "\n",
+      "Corruption type: brightness\n",
+      "Drift? No!\n",
+      "Feature-wise p-values:\n",
+      "[0.1102559]\n",
+      "Time (s) 26.567\n",
+      "\n",
+      "Corruption type: pixelate\n",
+      "Drift? Yes!\n",
+      "Feature-wise p-values:\n",
+      "[0.]\n",
+      "Time (s) 27.632\n"
+     ]
+    }
+   ],
+   "source": [
+    "make_predictions(cd, X_h0, X_c, corruption)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, drift was only detected on the corrupted datasets. The p-value is well above the $0.05$ threshold for *H0* and below for the corrupted datasets. This confirms that the corruptions impacted the certainty with which the model was able to predict, as we would expect."
```

Comment:
Mention here that no drift was detected on brightness as performance didn't really drop much.

Reply:
Ahh I had done this but my notebooks often don't save all the changes for some reason. Will redo